src/applications/debt-letters/components/DebtLettersWrapper.jsx

```diff
@@ -4,28 +4,60 @@ import LoadingIndicator from '@department-of-veterans-affairs/formation-react/Lo
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 import CallToActionWidget from 'platform/site-wide/cta-widget';
 import { bindActionCreators } from 'redux';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
 import { fetchDebtLetters, fetchDebtLettersVBMS } from '../actions';
 
 class DebtLettersWrapper extends Component {
   componentDidMount() {
-    this.props.fetchDebtLetters();
-    this.props.fetchDebtLettersVBMS();
+    if (this.props.showDebtLetters !== false) {
+      this.props.fetchDebtLetters();
+      this.props.fetchDebtLettersVBMS();
+    }
```

Comment:
This is the inverse of the reasoning for the change in componentDidMount - We cannot use falsey over false incase of an undefined which happens when visiting the route directly

Reply:
Isn't undefined falsey?

Reply:
@mr0sari0 yep - but we don't want to omit fetching on the first load because of the aforementioned issues with feature flips being undefined if a user visits a route directly